src/main/java/de/refactoringbot/refactoring/supportedrefactorings/AddOverrideAnnotation.java

```diff
@@ -35,7 +35,7 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 		CompilationUnit compilationUnit = LexicalPreservingPrinter.setup(JavaParser.parse(in));
 
 		MethodDeclaration methodDeclarationToModify = RefactoringHelper
-				.getMethodByLineNumberOfMethodName(issue.getLine(), compilationUnit);
+				.getMethodDeclarationByLineNumber(issue.getLine(), compilationUnit);
```

Comment:
@MarvinWyrich: I want to report a bug here. It's not at this line, its above. On line 32 we init the String "methodName" to "null". On line 54 we use that String to create a commit message, however we never set that string. I would remove "methodName" and replace on line 54 "methodName" with "methodDeclarationToModify.getNameAsString()"